pkg/scheduler/eventhandlers_test.go

```diff
@@ -334,3 +342,82 @@ func TestPreCheckForNode(t *testing.T) {
 		})
 	}
 }
+
+// Test for informers of resources we care about is registered
+func TestAddAllEventHandlers(t *testing.T) {
+	tests := []struct {
+		name              string
+		gvkMap            map[framework.GVK]framework.ActionType
+		expectInformerNum int
+	}{
+		{
+			name:              "default handlers in framework",
+			gvkMap:            map[framework.GVK]framework.ActionType{},
+			expectInformerNum: 2,
+		},
+		{
+			name: "add GVKs handlers defined in framework dynamically",
+			gvkMap: map[framework.GVK]framework.ActionType{
+				"Pod":                               framework.Add | framework.Delete,
+				"PersistentVolume":                  framework.Delete,
+				"storage.k8s.io/CSIStorageCapacity": framework.Update,
+			},
+			expectInformerNum: 4,
+		},
+		{
+			name: "add GVKs handlers defined in plugins dynamically",
+			gvkMap: map[framework.GVK]framework.ActionType{
+				"daemonsets.v1.apps": framework.Add | framework.Delete,
+				"cronjobs.v1.batch":  framework.Delete,
+			},
+			expectInformerNum: 4,
+		},
+		{
+			name: "add GVKs handlers defined in plugins dynamically, with one illegal GVK form",
+			gvkMap: map[framework.GVK]framework.ActionType{
+				"daemonsets.v1.apps":    framework.Add | framework.Delete,
+				"custommetrics.v1beta1": framework.Update,
+			},
+			expectInformerNum: 3,
+		},
+	}
+
+	for _, tt := range tests {
+		scheme := runtime.NewScheme()
```

Comment:
L386~LL396 can be moved out of the for loop (L385).

Reply:
ok